src/aslPaths.pegjs

```diff
@@ -119,7 +122,7 @@ comparison_op
 
 value
    = STRING
-   / NUMBER
+   / minus:MINUS? nbr:NUMBER { return (minus === '-' ? -1 : 1) * parseFloat(nbr)}
```

Comment:
this is why the original expression was rejected. There was no support for negative numbers as values.

Small change here to support plus the changes above to support different combinations of valid args for MathAdd